tuf/ngclient/updater.py

```diff
@@ -267,6 +268,7 @@ def download_target(
 
         if filepath is None:
             filepath = self._generate_target_file_path(targetinfo)
+        Path(filepath).parent.mkdir(exist_ok=True, parents=True)
```

Comment:
This is a case where Pathlib is much more ergonomic so I'm using it even though we don't use Paths generally